### DISCUSSION TITLE
please support table style such as excel. span cells

### DISCUSSION THREAD

**Discussion by jinhefeng** [toeverything/AFFiNE#6493] [General]
### Description

AFFiNE is great!
But I have lots of excel files. can they be supported? especially merge cells.

### Use case

almost everyone

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by CatsJuice** [comment]
Affine currently does not have a pure table. The current table-view (also known as a database) is actually a complex data structure, so implementing cell merging in it may be unrealistic. However, we might have plans for a pure table in the future? @hwangdev97

**Comment by hwangdev97** [comment]
It is not easy to do this well, and we do not yet have plans for Excel-like table functions

**Comment by houshasen** [comment]
Just a regular simple table alone would be great starting point.

**Comment by hannah23280** [comment]
I second this!!! 
This beat notion out of the water
